08-read-write-plot.qmd

```diff
@@ -291,6 +292,44 @@ The next two sections will demonstrate how to do this.
 
 ### Vector data
 
+The counterpart of `gpd.read_file` is the `.to_file` method that a `GeoDataFrame` has. It allows you to write `GeoDataFrame` objects to a wide range of geographic vector file formats, including the most common, such as `.geojson`, `.shp` and `.gpkg`. Based on the file name, `.to_file` decides automatically which driver to use. The speed of the writing process depends also on the driver.
+
+```{python}
+world.to_file('output/world.gpkg')
+```
+
+Note: if you try to write to the same data source again, the function will overwrite the file:
```

Comment:
Is there a way to changes this? In `sf` there is `delete_dsn = FALSE` by default I believe.

Reply:
I don't think there is anything similar to avoid overwriting using an argument (at least didn't find any)